src/components/ui/Badge/Badge.tsx

```diff
@@ -10,17 +12,41 @@ type Props = {
   small?: boolean
   variant?: BadgeVariants
   children: ReactNode
+  onClose?: () => void
+  interactive?: boolean
 }
 
-const Badge = ({ small = false, variant, children, ...otherProps }: Props) => {
+const Badge = ({
+  variant,
+  children,
+  onClose,
+  small = false,
+  interactive = false,
+  ...otherProps
+}: Props) => {
   return (
     <UIBadge
       className="badge"
       data-store-badge={small ? 'small' : ''}
       data-store-badge-variant={variant}
+      data-store-badge-interactive={interactive}
       {...otherProps}
     >
-      {children}
+      <span>{children}</span>
+      {(interactive || onClose) && (
```

Comment:
This way it generates a little inconsistency in the style, because if we add onClose and not add interactive we would have the button, but not the style, like in the gif:


![badge](https://user-images.githubusercontent.com/51174217/159057153-19305865-ce26-4e8e-8f31-bf3cb0ecbc20.gif)

Reply:
I think we have two options, show the button only when "interactive" is "pass" or add the data-attribute when onClose is pass too